src/browser/modules/Editor/ActionButtons.tsx

```diff
@@ -36,22 +40,28 @@ interface ActionButton {
 
 const ActionButtons = ({
   buttons,
-  width = 24
-}: ActionButtonProps): JSX.Element => (
-  <ActionButtonSection>
-    {buttons.map((btn: ActionButton) => (
-      <EditorButton
-        data-testid={`editor-${btn.title.split(' ')[0]}`}
-        onClick={btn.onClick}
-        disabled={btn.disabled}
-        title={btn.title}
-        icon={btn.icon}
-        key={`editor${btn.title}`}
-        color={btn.iconColor}
-        width={width}
-      />
-    ))}
-  </ActionButtonSection>
-)
+  width = 24,
+  editorValue
+}: ActionButtonProps): JSX.Element => {
+  return (
+    <>
```

Comment:
Are the return statement + fragment needed?

Reply:
yeah I dont think it does 👍